examples/binance-markout/dashboard/dashboard.css

```
@@ -0,0 +1,190 @@
+    --panel: #161b22;
+    --border: #30363d;
+    --fg: #c9d1d9;
+    --fg-dim: #7d8794;
+    --fg-faint: #4b5563;
+    --accent: #e6edf3;
+    --buy: #3fb950;
+    --buy-dim: #1a4e23;
+    --sell: #f85149;
+    --sell-dim: #5a1d1d;
+    --warn: #d29922;
+    --font-mono: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace;
```

Stylelint (17.7.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)